### PR TITLE
BUG: Fix interaction handle enable/disable not working in views

### PR DESCRIPTION
**Pull request by Sunderlandkyl**
When right clicking in a view to call "toggleInteractionBox()", the current subject hierarchy ID was sometimes invalid, causing nothing to happen. Fixed by getting the transform node from "getTransformNodeForAction()", which works in both SH widgets and in views.

Also ensures that all context menu widgets are updated in "showViewContextMenuActionsForItem()".

### CODE REVIEW

File: Modules/Loadable/Transforms/SubjectHierarchyPlugins/qSlicerSubjectHierarchyTransformsPlugin.cxx [hunk 1]
```diff
@@ -919,12 +919,25 @@ void qSlicerSubjectHierarchyTransformsPlugin::showViewContextMenuActionsForItem(
   d->showResetCenterOfTransformationForTransformedNodesActions(transformNode);
 
   vtkMRMLTransformDisplayNode* displayNode = transformNode ? vtkMRMLTransformDisplayNode::SafeDownCast(transformNode->GetDisplayNode()) : nullptr;
+
+  bool wasBlocked = d->ToggleInteractionAction->blockSignals(true);
```
Comment (lassoan) [inline Modules/Loadable/Transforms/SubjectHierarchyPlugins/qSlicerSubjectHierarchyTransformsPlugin.cxx:923]: Is there a specific reason for not using QSignalBlocker?

**Review by lassoan — APPROVED** [review]
Good catch, thank you!

File: Modules/Loadable/Transforms/SubjectHierarchyPlugins/qSlicerSubjectHierarchyTransformsPlugin.cxx (hunk 1 above)
Comment (Sunderlandkyl) [reply Modules/Loadable/Transforms/SubjectHierarchyPlugins/qSlicerSubjectHierarchyTransformsPlugin.cxx]: No specific reason, just force of habit. I've updated it to use QSignalBlocker.

**Review by lassoan — APPROVED** [review]
Thank you, it looks good.

**Pull request merged**